sig-and-wg/sig-core/proposals/event-bus-monitoring-metrics.md

```diff
@@ -0,0 +1,38 @@
+|middleware availability | Health monitor of middleware like NATS Streaming | Helps in troubleshooting system failures cases, troubleshooting events not delivered cases and system integrity|
+
+## Publish App Metrics
+
+|Metric Name |Description |Motivation |
+|------------|------------|-----------|
+|events in |Total Number of received events | Gives insights about system load|
+|events in per {namespace &\| source ID &\| event type &\| event type version} |Number of events requests per {namespace &\| source ID &\| event type &\| event type version} | Filtering received events by different criteria combination can help with giving a better insight about the system load as well as troubleshooting events not delivered cases and system integrity|
+|succeeded events per {namespace &\| source ID &\| event type &\| event type version} | Number of succeeded events per {namespace &\| source ID &\| event type &\| event type version} | Helps in troubleshooting events not delivered cases and system integrity as well as being a system health indicator|
+|ignored events per {namespace &\| source ID &\| event type &\| event type version} | Number of ignored events per {namespace &\| source ID &\| event type &\| event type version} | Helps in troubleshooting events not delivered cases and system integrity as well as being a system health indicator|
+|failed events per {namespace &\| source ID &\| event type &\| event type version} | Number of failed events per {namespace &\| source ID &\| event type &\| event type version} | Helps in troubleshooting events not delivered cases and system integrity as well as being a system health indicator|
+|latency 99 percentile | Latency 99 percentile of published event request | High latency maybe a reason for not delivered events in some cases and it gives an insight about the system health and performance
+
+## Push App Metrics
+
+|Metric Name |Description |Motivation |
+|------------|------------|-----------|
+|events in | Number of events push requests | Comparing this metric to the received events by the publish app metric can help in troubleshooting events not delivered cases and system integrity|
+|pushed messages | Number of pushed messages to all consumers | Giving insight about knative eventing health by monitoring the rates as well as assuring the system integrity|
+|latency 99 percentile to all consumers|Latency 99 percentile of pushed events to all consumers | Gives insight about the knative/events delivery system performance and can help in troubleshooting events not delivered scenarios|
+|latency 99 percentile to a single consumer | Latency 99 percentile of pushed message to a single consumer | Assessing the system delivery performance and contributes to judging the whole system performance diagnosis|
+|failed pushed messages | Number of failed pushed messages | Help in troubleshooting events not delivered scenarios and system integrity|
```

Comment:
I'm guessing you'll consider the delivery status code? 
https://github.com/kyma-project/kyma/blob/3c2c64319e867bdf6cbace9717c2457a9f1d0ddf/components/event-bus/internal/push/handlers/message_handler.go#L110

Reply:
```suggestion
|failed pushed messages | The number of failed pushed messages | Facilitates troubleshooting unsuccessful Event deliveries and system integrity issues|
```